src/views/pacbio/PacbioRunIndex.vue

```diff
@@ -185,7 +185,8 @@ export default {
       return import.meta.env.VITE_TRACTION_BASE_URL + '/v1/pacbio/runs/' + id + '/sample_sheet'
     },
     generateVersionColour(versionIndex) {
-      const colours = ['gray', 'red', 'yellow', 'green', 'blue', 'indigo', 'purple', 'pink']
+      // colours used in TractionBadge
+      const colours = ['red', 'orange', 'yellow', 'green', 'blue', 'indigo', 'purple', 'pink']
```

Comment:
Just a thought - maybe, we can move these colours to TractionBadge class itself and can be exported from there, just in case this need to be accessed in a similar way in other places in future